- child_frame_id: wx250s/base_link
  frame_id: camera_color_optical_frame
  qw: 1.0
  qx: 0.0
  qy: 0.0
  qz: 0.0
  x: 0.0
  y: 0.0
  z: 0.0
